Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -3598,6 +3598,20 @@ InvisibleAutoplayNotPermitted:
     WebCore:
       default: false
 
+InvokeAttributeEnabled:
+  type: bool
+  status: stable
```

Comment:
Shouldn't it be 'preview' for few releases?

Reply:
🤦 yes thank you!

Reply:
Or even just "testing" (I think that's the correct status) potentially?

Reply:
Yes - or even 'testable' till we have web-spec sorted out.